Scripts/cba.py

```diff
@@ -395,13 +412,17 @@ def calc_revenue(demands, costs):
     args = parser.parse_args()
     log.initialize(args)
     wb = load_workbook(os.path.join(SCRIPT_DIR, "CBA_kehikko.xlsx"))
-    run_cost_benefit_analysis(
+    results = run_cost_benefit_analysis(
         args.baseline_scenario, args.projected_scenario, 1, wb)
-    if args.baseline_scenario_2 is not None and args.baseline_scenario_2 != "undefined":
+    if (args.baseline_scenario_2 is not None
+            and args.baseline_scenario_2 != "undefined"):
         run_cost_benefit_analysis(
             args.baseline_scenario_2, args.projected_scenario_2, 2, wb)
```

Comment:
What about the results in this second CBA analysis?

Reply:
I think we can neglect them. Would need to rethink the whole structure, because per-day results for two different years cannot just be summed directly, they would need to be discounted.